docs/source/advanced.rst

```diff
@@ -415,22 +417,23 @@ For this reason, usage of :class:`podium.datasets.BucketIterator` is recommended
   >>> label = LabelField(name='label')
   >>> fields = {'text': text, 'label': label}
   >>>
-  >>> train, valid, test = SST.get_dataset_splits(fields=fields)
+  >>> sst_train, sst_valid, sst_test = SST.get_dataset_splits(fields=fields)
```

Comment:
This was intentionally `train, valid, test` as it should get replaced by the IMDB line from later on